akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowSplitWhenSpec.scala

```diff
@@ -263,17 +264,19 @@ class FlowSplitWhenSpec extends StreamSpec {
     }
 
     "fail substream if materialized twice" in assertAllStagesStopped {
+      @silent("deprecated")
```

Comment:
Is this still needed now that you're adding attributes below?

Reply:
No it is not, good catch!